sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/DoFnTester.java

```diff
@@ -239,7 +248,11 @@ public void finishBundle() {
     if (state == State.UNSTARTED) {
       startBundle();
     }
-    fnRunner.finishBundle();
+    try {
+      fn.finishBundle(createContext(fn));
+    } catch (Exception e) {
+      throw UserCodeException.wrap(e);
```

Comment:
`DoFnTester` is not expected to be run via a runner, but for directly unit testing a `DoFn`. So I don't think that it should wrap all these exceptions. Am I wrong?